devices/eurotronic/eurotronic_spzb0001_trv.json

```diff
@@ -0,0 +1,373 @@
+          },
+          "read": {
+            "at": "0x4002",
+            "cl": "0x0201",
+            "ep": 1,
+            "fn": "zcl:attr",
+            "mf": "0x1037"
+          },
+          "default": "none"
+        },
+        {
+          "name": "state/lastupdated"
+        },
+        {
+          "name": "state/on",
+          "refresh.interval": 660,
+          "read": {
+            "at": "0x0000",
+            "cl": "0x0006",
+            "ep": 1,
+            "fn": "zcl:attr"
+          },
+          "parse": {
+            "at": "0x0000",
+            "cl": "0x0006",
+            "ep": 1,
+            "eval": "Item.val = Attr.val",
+            "fn": "zcl:attr"
+          },
+          "default": false
+        },
+        {
+          "name": "state/temperature",
+          "refresh.interval": 660,
+          "read": {
+            "at": "0x0000",
+            "cl": "0x0201",
+            "ep": 1,
+            "fn": "zcl:attr"
+          },
+          "parse": {
+            "at": "0x0000",
+            "cl": "0x0201",
+            "ep": 1,
+            "eval": "Item.val = Attr.val",
+            "fn": "zcl:attr"
+          },
+          "default": 0
+        },
+        {
+          "name": "state/valve",
+          "refresh.interval": 660,
+          "read": {
+            "fn": "none"
+          },
+          "parse": {
+            "at": "0x0008",
+            "cl": "0x0201",
+            "eval": "Item.val = (Attr.val / 255) * 100;",
+            "fn": "zcl:attr"
+          },
+          "default": 0
+        },
+        {
+          "name": "state/windowopen",
+          "refresh.interval": 3660,
+          "parse": {
+            "at": "0x4008",
+            "cl": "0x0201",
+            "ep": 1,
+            "eval": "Item.val = (R.item('config/hostflags').val >> 3) & 0x01;",
+            "fn": "zcl:attr",
+            "mf": "0x1037"
+          },
+          "read": {
+            "fn": "none"
+          },
+          "default": false
+        }
+      ]
+    }
+  ],
+  "bindings": [
+    {
+      "bind": "unicast",
+      "src.ep": 1,
+      "cl": "0x0001",
+      "report": [
+        {
+          "at": "0x0021",
+          "dt": "0x20",
+          "min": 3600,
+          "max": 43200,
+          "change": "0x00000002"
+        }
+      ]
+    },
+    {
+      "bind": "unicast",
+      "src.ep": 1,
+      "cl": "0x0201",
+      "report": [
+        {
+          "at": "0x0000",
+          "dt": "0x29",
+          "min": 1,
+          "max": 600,
+          "change": "0x00000014"
+        },
+        {
+          "at": "0x0008",
+          "dt": "0x20",
+          "min": 1,
+          "max": 600,
+          "change": "0x00000001"
+        },
+        {
+          "at": "0x4002",
+          "dt": "0x18",
+          "mf": "0x1037",
+          "min": 1,
+          "max": 3600
+        },
+        {
+          "at": "0x4003",
+          "dt": "0x29",
+          "mf": "0x1037",
+          "min": 1,
+          "max": 600
```

Comment:
Legacy code has a reportable change of 50 (int value). Do we want to adapt that?